AutomatedTesting/src/test/resources/features/DeletePlanet.feature

```diff
@@ -0,0 +1,11 @@
+Feature: DeletePlanet
+
+    Scenario: Automated Delete Planet Test Case
+    Validating that Planets can be deleted by users no matter if the Planets have any orbiting Moons or not.
+
+        Given the User is on the <Home Page>
+        And the User enter  <planet name>
+        When clicks Delete Button
+        And the User see <alert message >
```

Comment:
More specific <alert message> would be "Then the User should see an alert saying '<'Delete Planet Result'>'"